docs/src/general/flavors-of-metal-stack.md

```diff
@@ -0,0 +1,21 @@
+# Flavors of metal-stack
+
+While metal-stack itself provides access to manage resources like machines, networks and ip addresses, it does not provide any higher abstractions on top when used on its own.
+
+As modern infrastructure and cloud native applications are designed with Kubernetes in mind, we provide two different layers on top of metal-stack to provide provisioning of clusters.
+
+## Plain
+
+Regardless which flavor of metal-stack you use, it is always possible to manually provision machines, networks and ip addresses. This is the most basic way of using metal-stack and is very similar to how traditional bare metal infrastructures are managed.
+
```

Comment:
Maybe we should mention that we did not really focus on plain metal-stack usage until lately. Firewall management and role management is therefore premature (reference MEP-4 and MEP-16).